justfile

```diff
@@ -3,6 +3,9 @@ set dotenv-load
 line_length := "100"
 coordinator_log_file := "$PWD/data/coordinator/regtest.log"
 maker_log_file := "$PWD/data/maker/regtest.log"
+pubspec := "$PWD/mobile/pubspec.yaml"
+testnet_coordinator := "COORDINATOR_P2P_ENDPOINT=038fb0eb9d9deeb4f0aa885d6c64abb0a26002871d2f7501349cd8d00f1f130045@195.201.81.94:9045"
+testnet_electrs := "ELECTRS_ENDPOINT=195.201.81.94:50000"
```

Comment:
I think you have these in your `.env`, so you could refer to them by `${VARIABLE}` notation instead of duplicating these

Reply:
I don't think the testnet configuration should be in the `.env` file - I think it can just be hardcoded for everybody.

Reply:
fair enough!